yt/frontends/halo_catalog/tests/test_outputs.py

```diff
@@ -0,0 +1,99 @@
+"""
+halo_catalog frontend tests
+
+
+
+"""
+
+#-----------------------------------------------------------------------------
+# Copyright (c) yt Development Team. All rights reserved.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+import numpy as np
+
+from yt.convenience import \
+    load as yt_load
+from yt.frontends.halo_catalog.data_structures import \
+    HaloCatalogDataset
+from yt.frontends.ytdata.utilities import \
+    save_as_dataset
+from yt.testing import \
+    assert_array_equal, \
+    TempDirTest
+from yt.units.yt_array import \
+    YTArray, \
+    YTQuantity
+
+def fake_halo_catalog(data):
+    filename = "catalog.0.h5"
+
+    ftypes = dict((field, '.') for field in data)
+    extra_attrs = {"data_type": "halo_catalog",
+                   "num_halos": data['particle_mass'].size}
+
+    ds = {'cosmological_simulation': 1,
+          'omega_lambda': 0.7,
+          'omega_matter': 0.3,
+          'hubble_constant': 0.7,
+          'current_redshift': 0,
+          'current_time': YTQuantity(1, 'yr'),
+          'domain_left_edge': YTArray(np.zeros(3), 'cm'),
+          'domain_right_edge': YTArray(np.ones(3), 'cm')}
+    save_as_dataset(
+        ds, filename, data,
+        field_types=ftypes, extra_attrs=extra_attrs)
+    return filename
+
+class HaloCatalogTest(TempDirTest):
+    def test_halo_catalog(self):
```

Comment:
These tests need to be decorated with `@requires_module('h5py')`

Reply:
Right, I always forget this. Thanks!